flink-python/pyflink/common/typeinfo.py

```diff
@@ -70,6 +72,18 @@ def __hash__(self) -> int:
     def __str__(self):
         return self._j_typeinfo.toString()
 
+    def need_conversion(self):
+        """
+        Does this type need to conversion between Python object and internal Wrapper object.
+        """
+        return False
+
+    def to_wrapper_type(self, obj):
```

Comment:
What about renamed to to_internal_type? The "Wrapper" prefix of the class name means wrapping the Java TypeInformation, however the functionality of "to_wrapper_type" means converting the Python Date/Time/Timestamp objects to int/long type. They are not the same thing and the name "wrapper" causes confusing.

Reply:
@dianfu OK. I would like to rename `to_wrapper_type` to `to_internal_type `.